index.html

```diff
@@ -1,28 +1,51 @@
+
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Tracom Academy website</title>
-</head>
-
-<body>
-    <header class="main-header">
-        <nav class="main-nav">
-            <img src="Assets/tracom_services_cover.jpg" alt="tracom-logo" class="main-logo">
-            <ul id="main-nav-ul">
-                <li><a href="#">Home</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Services</a></li>
-                <li><a href="#">Team</a></li>
-                <li><a href="#">Contact</a></li>
-            </ul>
-        </nav>
-    </header>
+    </head>
+    <body>
+        <header class="main-header">
+            <nav class="main-nav">
+                <img src="Assets/tracom_services_cover.jpg" alt="tracom-logo" class="main-logo" >
+                <ul id="main-nav-ul">
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Services</a></li>
+                    <li><a href="#">Team</a></li>
+                </ul>
+            </nav>            
+        </header>
 
-</body>
+    
+    
+        <div class="CON">
+            <h2>CONTACT FORM</h2>
+            
+            <form action="/CONTACT" method="get"> 
+             <label for="id">ID:</label>
```

Comment:
On contact form its very rare to ask the user his/her id, i suggest you go through some of the design online on contact page to see which fields are mostly used